### PR TITLE
Add PHP 8.0.2, 7.4.15, 7.3.27

### PR DESCRIPTION
**Pull request by mumumu**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,9 +22,9 @@ jobs:
           - '7.0.33'
           - '7.1.33'
           - '7.2.34'
-          - '7.3.26'
-          - '7.4.14'
-          - '8.0.1'
+          - '7.3.27'
+          - '7.4.15'
+          - '8.0.2'
         exclude:
```
Comment (GrahamCampbell) [inline .github/workflows/ci.yml:28]: Please update the exclude lists.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mumumu) [reply .github/workflows/ci.yml:28]: > Please update the exclude lists.

Should I add 7.0, 7.1, 7.2 which reached EOL?

Comment (GrahamCampbell) [reply .github/workflows/ci.yml:28]: That's not what I meant. You need replace all occurrences of `7.3.26` with `7.3.27` in this file, etc. Not just the first ones.

Comment (mumumu) [reply .github/workflows/ci.yml:28]: > Not just the first ones.

hmm. Currently, 7.3.x, 7.4.x, 8.0.x are not included in exclude list of ci.yml.

**Comment by GrahamCampbell** [comment]
Thanks.

**Pull request merged**